pykickstart/commands/repo.py

```diff
@@ -230,6 +230,7 @@ def _getParser(self):
                             of repos in this directory changes from release to
                             release and cannot be listed here. There will
```

Comment:
I think it is a good idea to guide the reader on how to get the list of the anaconda provided repo names.

But we can also notice the current values used by current Fedora releases ("fedora", "fedora-updates"), so not every reader needs to do the same task again, because we know the answer already. The instructions will useful to verify if the names are still valid, or to get the new one if Fedora had changed the names.

But I have also tried the (for me new) name "fedora-updates". I got an error, it doesn't work.
I have created a bug report https://bugzilla.redhat.com/show_bug.cgi?id=2186876 .
We may defer the change of the documentation until this bug is solved.